ospf6d/ospf6_neighbor.c

```diff
@@ -846,6 +846,39 @@ DEFPY(ipv6_ospf6_p2xp_neigh_poll_interval,
 
 	p2xp_unicast_hello_sched(p2xp_cfg);
 	return CMD_SUCCESS;
+
+}
+
+/* build state value */
+static void ospf6_neighbor_state_message(struct ospf6_neighbor *on,
```

Comment:
Can't we use static strings for this goal at all? Like:

```c
static const char *ospf6_neighbor_state_message(...) {
  if (on->ospf6_if->type == OSPF_IFTYPE_POINTOPOINT)
    return "PointToPoint";
  ...
}
```

Reply:
taken into account